app/controllers/web/my-account.js

```diff
@@ -37,7 +37,7 @@ async function update(ctx) {
       body[config.passport.fields.givenName];
     ctx.state.user[config.passport.fields.familyName] =
       body[config.passport.fields.familyName];
-    ctx.state.user.email = body.email;
+    ctx.state.user[config.userFields.fullEmail] = body.email;
```

Comment:
Don't use `fullEmail`, just use `email` as you had.  Full email includes Display Name too which we don't want to render here.  If you had other reasoning lmk